code/__DEFINES/~skyrat_defines/colors.dm

```diff
@@ -0,0 +1 @@
+#define LIGHT_COLOR_CLOCKWORK "#BE8700"
```

Comment:
I swear we have this already, make sure you move all color defines here

Reply:
We don't, not from what i've seen